stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -340,6 +340,12 @@ spec:
         storageClassName: "{{ .Values.persistentVolume.storageClass }}"
       {{- end }}
       {{- end }}
+      {{- if .Values.persistentVolume.selector }}
```

Comment:
Would prefer the `if `and `with` statements be combined into a single `with`, since `with` already is conditional

Reply:
Fixed